### ISSUE TITLE
I played 3 hours but I don't have halloween hat

### ISSUE THREAD

**Issue by jjuny768** [Uuvana-Studios/longvinter-windows-client#1273]
I played 3 hours after 1.0.9b update but I can't wear witch hat!
Also, in my steam inventory, I don't have witch hat item.

**Comment by esme82** [comment]
I also played for more than 3 hours and didn't get the yellow hat. Thank you for your time and help.

**Comment by Floffidinio** [comment]
Did you play for 3 hours without leaving the game? I only got the purple witch hat after playing for 1 hour straight without quitting the game

**Comment by esme82** [comment]
> Did you play for 3 hours without leaving the game? I only got the purple witch hat after playing for 1 hour straight without quitting the game

Yes, I played for more than 3hours and didn't get the yellow witch hat. I have it now though

**Comment by worthij92** [comment]
I also played for more than 3 hours on 3 consecutive days and still have not received the yellow hat. Actively playing as well, if that makes a difference. I spend most of my time on new islands so I wonder if being on the new islands doesnt count towards time?
